src/Core/Components/DataGrid/Columns/ColumnBase.razor

```diff
@@ -52,15 +52,37 @@
                     {
                         <FluentMenuItem OnClick="@(async () => await Grid.SortByColumnAsync(this))" @onkeydown="HandleSortMenuKeyDownAsync">
                             @GetSortOptionText()
+                            @if (Grid.ColumnSortLabels.Icon is not null)
+                            {
+                                <span style="display:flex; align-items: center;" slot="@Grid.ColumnSortLabels.Slot">
+                                    <FluentIcon Value="@(Grid.ColumnSortLabels.Icon)" Slot="@Grid.ColumnSortLabels.Slot" />
```

Comment:
Do you need to set the `slot` for the `span` and also for the `FluentIcon` ?
Same for other ResizeMenu and OptionsMenu below

Reply:
Yes it is a web component shortcoming. See https://fluentui-blazor.net/Menu#withicons